omnigibson/utils/transform_utils.py

```diff
@@ -1153,3 +1153,38 @@ def z_angle_from_quat(quat):
 def z_rotation_from_quat(quat):
     """Get the quaternion for the rotation around the Z axis produced by the quaternion."""
     return R.from_euler("z", z_angle_from_quat(quat)).as_quat()
+
+
+def integer_spiral_coordinates(n):
```

Comment:
is this for visualization? should this take in a scaling constant?

Reply:
It's for laying out scenes not just linearly but in a square to optimize max distance from origin. Currently disabled because with different scale scenes, the math gets tricky.